packages/slice-machine/src/modules/simulator/index.ts

```diff
@@ -206,96 +137,65 @@ function* checkSetupSaga(
       checkSimulatorSetup
     );
 
-    // All the backend checks are ok ask for the frontend Iframe check
-    if ("ok" === setupStatus.manifest && "ok" === setupStatus.dependencies) {
+    if (setupStatus.manifest === "ok") {
       yield put(
         checkSimulatorSetupCreator.success({
-          setupStatus: { iframe: null, ...setupStatus },
+          setupStatus,
         })
       );
-      yield put(connectToSimulatorIframeCreator.request());
-      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
-      const {
-        timeout,
-        iframeCheckKO,
-        iframeCheckOk,
-      }: {
-        iframeCheckOk: ReturnType<
-          typeof connectToSimulatorIframeCreator.success
-        >;
-        iframeCheckKO: ReturnType<
-          typeof connectToSimulatorIframeCreator.failure
-        >;
-        timeout: CallEffect<true>;
-      } = yield race({
-        iframeCheckOk: take(getType(connectToSimulatorIframeCreator.success)),
-        iframeCheckKO: take(getType(connectToSimulatorIframeCreator.failure)),
-        timeout: delay(10000),
-      });
-
-      if (iframeCheckOk && action.payload.callback) {
+      yield put(updateManifestCreator({ value: setupStatus.value }));
+      if (action.payload.callback) {
         action.payload.callback();
-        return;
-      }
-
-      if (timeout) {
-        yield put(connectToSimulatorIframeCreator.failure());
-        yield call(failCheckSetupSaga);
-        return;
-      }
-
-      if (iframeCheckKO) {
-        yield call(failCheckSetupSaga);
-        return;
       }
       return;
     }
-
-    // All the backend checks are ko and the request is coming from the "start"
-    const isTheFirstTime =
-      "ko" === setupStatus.manifest &&
-      "ko" === setupStatus.dependencies &&
-      action.payload.withFirstVisitCheck;
-
-    if (isTheFirstTime) {
-      yield put(openSetupDrawerCreator({}));
-      return;
-    }
-
-    yield put(
-      checkSimulatorSetupCreator.success({
-        setupStatus: { iframe: null, ...setupStatus },
-      })
-    );
     yield call(failCheckSetupSaga);
+    yield call(trackOpenSetupModalSaga);
   } catch (error) {
     // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
     yield put(checkSimulatorSetupCreator.failure(error as Error));
   }
 }
 
+function* connectToSimulatorIframe() {
+  yield put(connectToSimulatorIframeCreator.request());
+  const {
+    timeout,
```

Comment:
linting issue with `any`

Reply:
I'm unable to fix it, would you mind taking a look?